profiles/package.mask

```diff
@@ -17,6 +17,37 @@ dev-haskell/accelerate-io
 app-text/lit
 dev-haskell/highlighting-kate
 
+# Masked for removal. These require <dev-haskell/lens-4.19, but we're bumping
```

Comment:
Commit message usually starts from a file or a directory that is changed, like `profiles/package.mask: ...`.

Reply:
Got it, I will plan on rewording all my commits to match your earlier comment.